RiotSwiftUI/Modules/LocationSharing/LiveLocationSharingViewer/Test/Unit/LiveLocationSharingViewerViewModelTests.swift

```diff
@@ -30,4 +31,17 @@ class LiveLocationSharingViewerViewModelTests: XCTestCase {
         viewModel = LiveLocationSharingViewerViewModel(mapStyleURL: BuildSettings.defaultTileServerMapStyleURL, service: service)
         context = viewModel.context
     }
+    
+    func testIsUserBeingShared() {
+        XCTAssertTrue(context.viewState.isCurrentUserShared)
+    }
+    
+    func testToggleShowUserLocation() {
+        let service = MockLiveLocationSharingViewerService(currentUserSharingLocation: false)
+        let viewModel = LiveLocationSharingViewerViewModel(mapStyleURL: BuildSettings.defaultTileServerMapStyleURL, service: service)
+        XCTAssertFalse(viewModel.context.viewState.isCurrentUserShared)
+        XCTAssertFalse(viewModel.context.viewState.showsUserLocation)
+        viewModel.context.send(viewAction: .showUserLocation)
+        XCTAssertTrue(viewModel.context.viewState.showsUserLocation)
+    }
```

Comment:
Are we testing that the button is hidden when the map is from the current user in live sharing mode?